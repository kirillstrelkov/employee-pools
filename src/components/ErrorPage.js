import {Typography} from "@mui/material";
import React from "react";

function ErrorPage() {
  return <Typography variant="h5">Oops something went wrong...</Typography>;
}

export default ErrorPage;
