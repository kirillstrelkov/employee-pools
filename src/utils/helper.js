const LOCALE = "en-US";

export function formatDate(timestamp) {
  const d = new Date(timestamp);
  const time = d.toLocaleTimeString(LOCALE);
  return time.substr(0, 5) + time.slice(-2) + " | " + d.toLocaleDateString();
}

export function formatFloat(float) {
  return float.toFixed(2).toLocaleString(LOCALE);
}

export function sortQuestions(questions) {
  return questions.sort((a, b) => {
    return b.timestamp - a.timestamp;
  });
}
