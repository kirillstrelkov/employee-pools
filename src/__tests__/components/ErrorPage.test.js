import ErrorPage from "../../components/ErrorPage";
import {renderWithProviders} from "../utils";

describe("ErrorPage", () => {
  it("show apage", async () => {
    const view = renderWithProviders(<ErrorPage />);
    expect(view.baseElement).toMatchSnapshot();
  });
});
