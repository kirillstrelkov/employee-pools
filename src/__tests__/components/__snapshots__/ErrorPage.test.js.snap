// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage show apage 1`] = `
<body>
  <div>
    <h5
      class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
    >
      Oops something went wrong...
    </h5>
  </div>
</body>
`;
